packages/rocketchat-file-upload/lib/FileUploadBase.js

```diff
@@ -28,7 +28,7 @@ FileUploadBase = class FileUploadBase {
 	}
 
 	getFileName() {
-		return this.meta.name;
+		return this.meta.name.substring(0, 50);
```

Comment:
I kinda worry about just doing this, as this happens server side too.  Might be better to use something client side to keep it from overflowing?